src/ert/cli/main.py

```diff
@@ -44,6 +44,15 @@ def run_cli(args):
     os.chdir(res_config.config_path)
     ert = EnKFMain(res_config)
     facade = LibresFacade(ert)
+    if not facade.have_observations and args.mode not in [
+        ENSEMBLE_EXPERIMENT_MODE,
+        TEST_RUN_MODE,
+    ]:
+        raise RuntimeError(
+            f"To run {args.mode}, observations are needed.\n"
+            f" Please add an observation file to {args.config} example:\n"
```

Comment:
Very very minor - but i think it reads better with `Please add an observation file to {args.config}. Example: `